src/lxc/criu.c

```diff
@@ -742,8 +733,9 @@ static void do_restore(struct lxc_container *c, int status_pipe, struct migrate_
 		}
 
 		if (WIFEXITED(status)) {
+			char buf[4096];
```

Comment:
We use the buffer in the other if branch too, so I'm just re-using the same buffer.